src/core/surface/completion_queue.h

```diff
@@ -51,6 +52,16 @@ typedef struct grpc_cq_completion {
   gpr_uintptr next;
 } grpc_cq_completion;
 
+/** An alarm associated with a completion queue. */
```

Comment:
If this new cq alarms take the grpc_alarm name, what should the current grpc_alarm from alarm.h be called?